packages/mobile/src/components/details-tile/CollectionMetadata.tsx

```diff
@@ -0,0 +1,58 @@
+import { useGetPlaylistById, useGetCurrentUserId } from '@audius/common/api'
+import { useCollectionMetadata } from '@audius/common/hooks'
+import type { ID, Track } from '@audius/common/models'
+import {
+  cacheCollectionsSelectors,
+  type CommonState
+} from '@audius/common/store'
+import { useSelector } from 'react-redux'
+
+import { Flex } from '@audius/harmony-native'
+
+import { MetadataRow } from './MetadataRow'
+
+const { getCollectionTracks } = cacheCollectionsSelectors
+
+type CollectionMetadataProps = {
+  collectionId: ID
+}
+
+/**
+ * The additional metadata shown at the bottom of the Collection Screen Headers
+ */
+export const CollectionMetadata = ({
+  collectionId
+}: CollectionMetadataProps) => {
+  const { data: currentUserId } = useGetCurrentUserId({})
+  const { data: collection } = useGetPlaylistById(
+    { playlistId: collectionId!, currentUserId },
+    { disabled: !collectionId }
+  )
+  const collectionTracks = useSelector((state: CommonState) =>
```

Comment:
nit but maybe make do the duration calculation in the selector, so `const collectionDuration = useSelector(....)`?